src/main/resources/archetype-resources/src/main/java/EntryPoint.java

```diff
@@ -0,0 +1,5 @@
+package ${package};
+
+public interface EntryPoint {
```

Comment:
no need for this at all, unless trying to look like compatibility with gwt2, in which case the package should be com.google.gwt.core.client

Reply:
fixed